### PR TITLE
Updating Platform9 information

### PR DESCRIPTION
**Pull request by akshaisarathy**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6560)

### CODE REVIEW

**Comment by akshaisarathy** [comment]
How can I get my entries under "Turn-key Cloud Solutions", "Custom Solutions/On-Premise VMs" and "Custom Solutions/On-Premise VMs/Bare Metal"? I don't see "On-Premises turnkey cloud solutions" on the toc on the left.

**Comment by akshaisarathy** [comment]
CLA has been signed

**Comment by akshaisarathy** [comment]
@a-mccarthy : could you please review?

**Comment by akshaisarathy** [comment]
@surajssd @potterhe @alexcontini : please review

**Comment by heckj** [comment]
@akshaisarathy  I'm doing a pass through all the older open PRs and see that this hasn't received a lot of review attention. I'm afraid that it needs a rebase at this point as well, so I wanted to request you do that. I've assigned the PR to review to myself and will do a content review on it, but we'll need that rebase before we can merge it.

/assign

File: docs/getting-started-guides/platform9/platform9-cloud.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+approvers:
```
Comment (heckj) [inline docs/getting-started-guides/platform9/platform9-cloud.md]: please change this from `approvers` to `reviewers`

File: docs/getting-started-guides/platform9/platform9-cloud.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Review the Summary screen and click **CREATE CLUSTER**.
+
+<img src="../kubernetes-aws-summary.png">
+
+You can monitor the status of your cluster by clicking on the cluster name in the **Clusters** tab. Next, click on **Nodes** and look for the **Connected** status. This can take a few minutes.
+
+<img src="../kubernetes-aws-complete.png" width="50%" height="50%">
+
+The Kubernetes cluster is now ready for use. 
+
```
Comment (heckj) [inline docs/getting-started-guides/platform9/platform9-cloud.md]: please update this file to use one of the templates - see https://kubernetes.io/docs/home/contribute/page-templates/, the other getting started guides frequently use task or concept templates

File: docs/getting-started-guides/platform9/platform9-cloud.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+* TOC
+{:toc}
+
+
+## Introduction
+
+Through the following tutorial, you will learn to install [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/?utm_source=k8sio) on Amazon Web Services. Platform9 Managed Kubernetes provides auto-deploy functionality on AWS, making it easy for you to install Kubernetes.
+
+Auto-deploy for Google Cloud Platform and Microsoft Azure will be available in 2018. For installation on other clouds and on-premises, please follow the steps for [manual installation](kubernetes.io/platform9/platform9-on-prem/). Please reach us at support@platform9.com if you need an alternative installer.
```
Comment (heckj) [inline docs/getting-started-guides/platform9/platform9-cloud.md]: the link for manual installation doesn't look like it's correct

File: docs/getting-started-guides/platform9/platform9-on-prem.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+approvers:
```
Comment (heckj) [inline docs/getting-started-guides/platform9/platform9-on-prem.md]: please change 'approvers' to 'reviewers'

File: docs/getting-started-guides/platform9/platform9-on-prem.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Continue in the **Nodes** tab. Select the node(s) that you want to join the cluster by clicking the corresponding checkboxes. In this example, only one node is present and it is named `kubernetes-master1`. 
+
+Select **Attach To Cluster** and specify the cluster name, `devops`. 
+
+You can monitor the status of your cluster’s node(s) by clicking on the cluster name in the **Clusters** tab. Next, click on **Nodes** and look for the **Connected** status. This can take a few minutes.
+
+<img src="../kubernetes-node1-onprem-complete.png" width="50%" height="50%">
+
+Your Kubernetes cluster is now ready for use. Please follow the same steps above to add additional nodes to the cluster.
```
Comment (heckj) [inline docs/getting-started-guides/platform9/platform9-on-prem.md]: please update this file to use one of the templates - see https://kubernetes.io/docs/home/contribute/page-templates/, the other getting started guides frequently use task or concept templates

File: docs/tasks/job/parallel-processing-expansion.md [hunk 6]
```diff
@@ -42,7 +42,7 @@ Next, expand the template into multiple files, one for each item to be processed
 $ mkdir ./jobs
 $ for i in apple banana cherry
 do
-  cat job.yaml.txt | sed "s/\$ITEM/$i/" > ./jobs/job-$i.yaml
+  cat job.yaml | sed "s/\$ITEM/$i/" > ./jobs/job-$i.yaml
```
Comment (heckj) [inline docs/tasks/job/parallel-processing-expansion.md]: These changes seem very orthogonal to the rest of the comment, did you intend to  include them?

**Comment by chenopis** [comment]
@akshaisarathy Please rebase this PR. Thx

**Comment by zacharysarah** [comment]
👋 We've migrated the site's framework to Hugo. Please re-open this PR with changes to files in their post-migration location (`/content/en/docs/...`).

**Pull request closed without merging**